server/package.json

```diff
@@ -24,6 +24,10 @@
 		"./browser": {
 			"types": "./lib/browser/main.d.ts",
 			"browser": "./lib/browser/main.js"
+		},
+		"./utils": {
```

Comment:
Why do we need this. It will export this from the npm package and will force use to keep it stable as API.